monai/bundle/config_parser.py

```diff
@@ -383,6 +387,29 @@ def _do_parse(self, config: Any, id: str = "") -> None:
         else:
             self.ref_resolver.add_item(ConfigItem(config=item_conf, id=id))
 
+    @staticmethod
+    def check_key_duplicates(ordered_pairs: Sequence[tuple[Any, Any]]) -> dict[Any, Any]:
```

Comment:
@wyli Should we put this function in the `utils`? I feel it's not part of `ConfigParser` signature.

Thanks.

Reply:
done